apps/docs/src/app/platform/component-docs/platform-object-status/platform-object-status-docs.component.html

```diff
@@ -0,0 +1,86 @@
+<fd-docs-section-title [id]="'default'" [componentName]="'infolabel'">
```

Comment:
@Lokanathan-k please remove all unnecessary square brackets bindings, all over this PR
```suggestion
<fd-docs-section-title id="default" componentName="infolabel">
```